kubernetes/lib/kubernetes/release/changelog.rb

```
@@ -0,0 +1,87 @@
+        if require_empty_unreleased && has_unreleased && !unreleased_empty?
+          errors << "Unreleased section must be empty before cutting #{expected_tag}"
+        end
+
+        raise Error, errors.join("\n") unless errors.empty?
+
+        true
+      end
+
+      private
+
+      def entry(name)
```

RuboCop (1.85.1)

[convention] Use 2 (not 0) spaces for indented_internal_methods indentation.

(Layout/IndentationWidth)